packages/jest-config/src/__tests__/normalize.test.js

```diff
@@ -951,6 +956,32 @@ describe('preset', () => {
       ['a', 'aa'],
     ]);
   });
+
+  test('merges with options and transform preset is overridden by options', () => {
+    // Object initializer not used for properties as a workaround for
+    //  sort-keys eslint rule while specifying properties in
+    //  non-alphabetical order for a better test
+    const transform = {};
```

Comment:
You can just disable the rule for this object

Reply:
I shamelessly copied from the other test, but it makes sense what you said. I will update.

Reply:
Updated 👍